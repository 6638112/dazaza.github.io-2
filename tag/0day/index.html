<!doctype html><html lang="zh-hans"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>#0day</title><link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"><link rel="stylesheet" href="/img/css.css?random="><link data-rh="true" rel="icon" href="/img/favicon.ico"/><script data-ad-client="ca-pub-6067137220025946" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script><script type="text/javascript" src="https://platform-api.sharethis.com/js/sharethis.js#property=5effb96910009800120b8d4d&product=inline-share-buttons" async="async"></script>
<script>var _hmt = _hmt || [];(function() {var hm = document.createElement("script");hm.src = "https://hm.baidu.com/hm.js?03c1a0f31299b4a2fbb83c34d6beaac9";var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(hm, s);})();</script></head><body><div id="my_header"><div class="container"><nav class="navbar navbar-expand-lg"><a class="navbar-brand" href="/"><img alt="diglog" src="/img/logo.v1.gif" class="rounded-sm"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNavAltMarkup"><div class="navbar-nav"></div></div></nav></div></div><div class="container"><div id="my_content"><h1>#0day</h1><div class="row"><div class="col-lg-8 col-12"><div id="list"><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1041620.html">Windows 0day特权升级仍未修复 </a></div><div class="item_title_en"><a target="_blank" href="https://bugs.chromium.org/p/project-zero/issues/detail?id=2096">Windows 0day privilege escalation still not fixed</a><span>(bugs.chromium.org)</span></div><span class="my_story_list_date">2020-12-24 21:50</span><div class="my_story_list_item_desc">对单轨电车的反馈 </div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/0day/">#0day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/windows/">#windows</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/特权/">#特权</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/privilege/">#privilege</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/电车/">#电车</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1035632.html">0day“在野外”</a></div><div class="item_title_en"><a target="_blank" href="https://googleprojectzero.blogspot.com/p/0day.html">0day “In the Wild”</a><span>(googleprojectzero.blogspot.com)</span></div><span class="my_story_list_date">2020-11-21 20:48</span><div class="my_story_list_item_desc">零项目团队的任务是“使零日工作变得艰难”，即使发现和利用安全漏洞的成本更高。我们主要通过执行自己的安全性研究来实现这一目标，但有时我们还会研究“在野外”发现的零日漏洞的外部实例。这些案例以真实的方式很好地增强了我们从自己的研究中获得的见解，从而提供了对现实世界中攻击者行为和功能的有趣观察。
今天，我们将共享针对检测到......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/0day/">#0day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/day/">#day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/漏洞/">#漏洞</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1032422.html">谷歌的Project Zero披露Windows 0day一直处于活跃利用状态</a></div><div class="item_title_en"><a target="_blank" href="https://arstechnica.com/information-technology/2020/10/googles-project-zero-discloses-windows-0day-thats-been-under-active-exploit/?comments=1">Google’s Project Zero discloses Windows 0day that’s been under active exploit</a><span>(arstechnica.com)</span></div><span class="my_story_list_date">2020-10-31 8:47</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1032422.html"><img src="http://img2.diglog.com/img/2020/10/thumb_47c0eb942a9e144374ff55f768138d34.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">谷歌的Zero项目表示，黑客一直在积极利用Windows Zeroday，该漏洞几乎要到两周后才能打上补丁。
为了与长期的政策保持一致，谷歌的漏洞研究小组给了微软7天的最后期限来修复安全漏洞，因为它正在被积极利用。通常，Project Zero会在90天后或补丁程序可用时(以先到者为准)披露漏洞。
在跟踪漏洞时，CV......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/0day/">#0day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/windows/">#windows</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/project/">#project</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1032369.html">谷歌的Project Zero披露Windows 0day一直处于活跃利用状态</a></div><div class="item_title_en"><a target="_blank" href="https://arstechnica.com/information-technology/2020/10/googles-project-zero-discloses-windows-0day-thats-been-under-active-exploit/">Google’s Project Zero discloses Windows 0day that’s been under active exploit</a><span>(arstechnica.com)</span></div><span class="my_story_list_date">2020-10-31 4:0</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1032369.html"><img src="http://img2.diglog.com/img/2020/10/thumb_47c0eb942a9e144374ff55f768138d34.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">谷歌的Zero项目表示，黑客一直在积极利用Windows Zeroday，该漏洞几乎要到两周后才能打上补丁。
为了与长期的政策保持一致，谷歌的漏洞研究小组给了微软7天的最后期限来修复安全漏洞，因为它正在被积极利用。通常，Project Zero会在90天后或补丁程序可用时(以先到者为准)披露漏洞。
在跟踪漏洞时，CV......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/0day/">#0day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/windows/">#windows</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/project/">#project</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1024503.html">ToR 0day：查找IP地址</a></div><div class="item_title_en"><a target="_blank" href="https://www.hackerfactor.com/blog/index.php?/archives/896-Tor-0day-Finding-IP-Addresses.html">Tor 0day: Finding IP Addresses</a><span>(www.hackerfactor.com)</span></div><span class="my_story_list_date">2020-9-17 12:12</span><div class="my_story_list_item_desc">去年2月，我的Tor洋葱服务遭遇了基于Tor的分布式拒绝服务(DDoS)攻击。我花了几天时间分析攻击，开发缓解方案，并保护我的服务器。(我为互联网档案馆(Internet Archive)运行的Tor服务停机了几个小时，但在攻击的大部分时间里，我设法保持了它的正常运行。)。在试图找到创造性的方法来维持服务的同时，我咨......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/0day/">#0day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/查找/">#查找</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/finding/">#finding</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/tor/">#tor</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1017749.html">0天，一个失败的补丁，一个后门威胁。周二更新亮点</a></div><div class="item_title_en"><a target="_blank" href="https://arstechnica.com/information-technology/2020/08/update-tuesday-fixes-2-0days-and-botched-patch-for-a-backdoor-threat/">0days, a failed patch, and a backdoor threat. Update Tuesday highlights</a><span>(arstechnica.com)</span></div><span class="my_story_list_date">2020-8-12 22:27</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1017749.html"><img src="http://img.diglog.com/img/2020/8/thumb_bd0ce3f44c19a35889771319d15cac78.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">周二，微软修补了120个漏洞，其中两个值得注意，因为它们受到了积极的攻击，第三个漏洞是因为它修复了之前的一个安全漏洞补丁，该漏洞允许攻击者获得一个即使在机器更新后仍然存在的后门。
零日漏洞之所以得名，是因为受影响的开发人员在安全漏洞受到攻击之前有零天的时间发布补丁。零日漏洞攻击可能是最有效的攻击之一，因为它们通常不会......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/0day/">#0day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/失败/">#失败</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/tuesday/">#tuesday</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/漏洞/">#漏洞</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1013798.html">ToR 0day：停止ToR连接</a></div><div class="item_title_en"><a target="_blank" href="https://www.hackerfactor.com/blog/index.php?/archives/888-Tor-0day-Stopping-Tor-Connections.html">Tor 0day: Stopping Tor Connections</a><span>(www.hackerfactor.com)</span></div><span class="my_story_list_date">2020-7-24 1:16</span><div class="my_story_list_item_desc">当遇到安全漏洞时，我有一个基本的理念：尽最大努力将其报告给合适的人。有时候，报道是毫无痛苦的。通常情况下，这有点挑战性。十多年前，我试图向Verisign报告一个问题。一名安全工作人员花了数周时间不断纠缠，才将漏洞传递到链条上。他的经理看到了这个问题，在电话里感谢了我，并给我寄了一盒礼物，以表示我的努力得到了赞赏。(......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/0day/">#0day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/tor/">#tor</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1000690.html">用iOS 0day盗取您的短信</a></div><div class="item_title_en"><a target="_blank" href="https://wojciechregula.blog/post/stealing-your-sms-messages-with-ios-0day/">Stealing your SMS messages with iOS 0day</a><span>(wojciechregula.blog)</span></div><span class="my_story_list_date">2020-5-5 11:18</span><div class="my_story_list_item_desc">这是一个特别的帖子，因为我完全基于另一位研究人员，s1guza的0day。所有的故事都是从下面的推特开始的：Siguza告诉我们，他的0day在iOS13.5Beta3中打了补丁。因此，对于最新的非测试版iOS(13.4.1)来说，这实际上是一次沙盒逃脱。在这篇文章中，我将向您展示我如何复制该错误并编写恶意应用程序，......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/0day/">#0day</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/ios/">#ios</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/盗取/">#盗取</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/sms/">#sms</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/应用程序/">#应用程序</a></button></div></div></div></div><div class="col-lg-4 col-0"><div class="my_movie_list_item shadow p-3 mb-5 bg-white rounded"><button type="button" class="btn btn-link my_tag"><a href="/tag/美国/">#美国</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/程序/">#程序</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/rust/">#rust</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/数据/">#数据</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/warning/">#warning</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/代码/">#代码</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/ai/">#ai</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/软件/">#软件</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/linux/">#linux</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/设计/">#设计</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/学习/">#学习</a></button></div><div class="my_movie_list_item shadow p-3 mb-5 bg-white rounded"><button type="button" class="btn btn-link my_tag"><a href="/tag/web2.0/">#web2.0</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/设计/">#设计</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/创意/">#创意</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/摄影/">#摄影</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/图片/">#图片</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/软件/">#软件</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/视频/">#视频</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/手机/">#手机</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/广告/">#广告</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/iphone/">#iphone</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/网站/">#网站</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/免费/">#免费</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/下载/">#下载</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/windows/">#windows</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/微软/">#微软</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/firefox/">#firefox</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/苹果/">#苹果</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/blog/">#blog</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/音乐/">#音乐</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/博客/">#博客</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/wordpress/">#wordpress</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/恶搞/">#恶搞</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/艺术/">#艺术</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/qq/">#qq</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/web/">#web</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/工具/">#工具</a></button></div></div></div></div><div id="my_footer"><div class=""><a href="/tags/">tags</a> <a href="/users/">users</a></div>&copy;2012-2021 diglog.com </div></div></body></html>