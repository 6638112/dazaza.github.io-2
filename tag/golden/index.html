<!doctype html><html lang="zh-hans"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>#golden</title><link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"><link rel="stylesheet" href="/img/css.css?random="><link data-rh="true" rel="icon" href="/img/favicon.ico"/><script data-ad-client="ca-pub-6067137220025946" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script><script type="text/javascript" src="https://platform-api.sharethis.com/js/sharethis.js#property=5effb96910009800120b8d4d&product=inline-share-buttons" async="async"></script>
<script>var _hmt = _hmt || [];(function() {var hm = document.createElement("script");hm.src = "https://hm.baidu.com/hm.js?03c1a0f31299b4a2fbb83c34d6beaac9";var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(hm, s);})();</script></head><body><div id="my_header"><div class="container"><nav class="navbar navbar-expand-lg"><a class="navbar-brand" href="/"><img alt="diglog" src="/img/logo.v1.gif" class="rounded-sm"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNavAltMarkup"><div class="navbar-nav"></div></div></nav></div></div><div class="container"><div id="my_content"><h1>#golden</h1><div class="row"><div class="col-lg-8 col-12"><div id="list"><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1062509.html">金手铐 </a></div><div class="item_title_en"><a target="_blank" href="https://avc.com/2021/05/golden-handcuffs-2/">Golden Handcuffs</a><span>(avc.com)</span></div><span class="my_story_list_date">2021-5-14 1:21</span><div class="my_story_list_item_desc">在整个初创公司和技术领域的基于股票的补偿计划基于“金色手铐” - 员工不能离开的想法，因为如果他们这样做，他们会放弃太多钱。
 我从来没有喜欢那个概念。感觉就像留在孩子们的坏婚姻一样。
 所以我一直参与重新思考的一项努力，这种练习和其中一个努力在本周早些时候在董事会和赔偿委员会在董事会和赔偿委员会上发光，博客有关他们......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/手铐/">#手铐</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/golden/">#golden</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/员工/">#员工</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1041495.html">考虑一下黄金痣（2019） </a></div><div class="item_title_en"><a target="_blank" href="https://www.lrb.co.uk/the-paper/v41/n08/katherine-rundell/consider-the-golden-mole">Consider the Golden Mole (2019)</a><span>(www.lrb.co.uk)</span></div><span class="my_story_list_date">2020-12-23 22:41</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1041495.html"><img src="http://img2.diglog.com/img/2020/12/thumb_dfb1a05e3a22621c3dceca07acb8e037.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">Warning: Can only detect less than 5000 characters</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/黄金/">#黄金</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/golden/">#golden</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/warning/">#warning</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1039818.html">拥有黄金机票的人（2019） </a></div><div class="item_title_en"><a target="_blank" href="https://narratively.com/the-man-with-the-golden-airline-ticket/">The Man with the Golden Airline Ticket (2019)</a><span>(narratively.com)</span></div><span class="my_story_list_date">2020-12-15 21:33</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1039818.html"><img src="http://img2.diglog.com/img/2020/12/thumb_b8dc8e3497f44fe8c8496bbd536e6477.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">2009年3月10日，我在伊利诺伊州北区的美国巡回法院提起诉讼。 Rothstein诉American Airlines，Inc.为我的父亲，原告人Steven Rothstein和被告（当时世界第三大航空公司）担任主演。美国航空的年收入为230亿美元，没有损失。对我父亲而言，这是挽救他生命的最后努力。
 这就是一切......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/黄金/">#黄金</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/golden/">#golden</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/父亲/">#父亲</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1032448.html">为什么我们生活在历史小说的黄金时代？(2019年)</a></div><div class="item_title_en"><a target="_blank" href="https://www.nytimes.com/2019/05/07/t-magazine/historical-fiction-books.html">Why Are We Living in a Golden Age of Historical Fiction? (2019)</a><span>(www.nytimes.com)</span></div><span class="my_story_list_date">2020-10-31 12:47</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1032448.html"><img src="http://img2.diglog.com/img/2020/10/thumb_1d1a4fbcc6a721b400334498089ab83b.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">在2013年的一次采访中，作家乔纳森·弗兰岑(Jonathan Franzen)的小说被誉为当代美国中产阶级生活的“画窗”。当他承认自己抵制历史小说时，他表达了许多读者的感受，解释说他被那些“致力于让自己的生活和他们发现自己所处的世界有意义”的作家所吸引，这些作家明显参与了这场游戏。这是一种很难不同意的观点--把过去......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/历史/">#历史</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/golden/">#golden</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1025537.html">四堂金色的课(2003)</a></div><div class="item_title_en"><a target="_blank" href="https://www.nature.com/articles/426389a">Four Golden Lessons (2003)</a><span>(www.nature.com)</span></div><span class="my_story_list_date">2020-9-22 19:20</span><div class="my_story_list_item_desc">大约一百年前，当我获得本科学位时，物理文献对我来说就像是一片广阔的、未被探索的海洋，在开始自己的任何研究之前，我必须先绘制其中的每一部分。我怎么能在不知道所有已经做过的事情的情况下做任何事情呢？幸运的是，在我读研究生的第一年，我很幸运地落入了资深物理学家的手中，他们不顾我的焦虑反对，坚持要求我必须开始研究，并在进行过......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/金色/">#金色</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/golden/">#golden</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/科学/">#科学</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1020048.html">黄金时代 / 鼎盛时期</a></div><div class="item_title_en"><a target="_blank" href="https://gaby.dev/posts/golden-age">Golden Age</a><span>(gaby.dev)</span></div><span class="my_story_list_date">2020-8-25 1:15</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1020048.html"><img src="http://img.diglog.com/img/2020/8/thumb_0323cf6a2fce7a7c526645767b4f9a11.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">他把黑色毛圈汗带系在额头上，小心翼翼地不惊动平坦的仙台皮垫。他闭上了眼睛。一个灰色的圆盘，千叶天空的颜色。圆盘开始旋转，变得更快，变成一个灰白色的球体。不断扩展--为他开花，流畅的霓虹灯折纸技巧，展开他无边的家，他的国家，透明的3D棋盘延伸到无限。向东海岸裂变管理局的阶梯状深红色金字塔的内眼张开，在美国三菱银行的绿色......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/黄金时代/">#黄金时代</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/golden/">#golden</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/网络空间/">#网络空间</a></button></div></div><div class="my_story_list_item shadow p-3 mb-5 bg-white rounded clearfix"><div class="item_title"><a target="_blank" href="/story/1017350.html">这会是健身新黄金时代的开始吗？</a></div><div class="item_title_en"><a target="_blank" href="https://www.forbes.com/sites/markhall/2020/08/10/golden-age-of-fitness/#44fc16b22dff">Could This Be the Start of a New Golden Age of Fitness?</a><span>(www.forbes.com)</span></div><span class="my_story_list_date">2020-8-10 22:53</span><div class="float-md-right my_story_img_thumb"><a target="_blank" href="/story/1017350.html"><img src="http://img.diglog.com/img/2020/8/thumb_75e69b8a931580a7e52fda7f62dcb7dc.jpg" class="img-fluid my_story_img_thumb" onerror="this.style.display='none'"></a></div><div class="my_story_list_item_desc">在过去的几十年里，越来越多的美国人接受了更健康的生活方式，其中包括更好的营养和定期锻炼。
数据显示，人们的优先事项已经发生了一些变化，以反映出对自我护理和健康的更高重要性。
2019年，据估计，超过20%的美国人属于健身房、健身房或健身俱乐部。事实上，据报道，自2008年以来，健身俱乐部会员人数增加了37%。
与此同......</div><div class="my_item_tag_container"><button type="button" class="btn btn-light my_tag"><a href="/tag/健身/">#健身</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/golden/">#golden</a></button><button type="button" class="btn btn-light my_tag"><a href="/tag/健身房/">#健身房</a></button></div></div></div></div><div class="col-lg-4 col-0"><div class="my_movie_list_item shadow p-3 mb-5 bg-white rounded"><button type="button" class="btn btn-link my_tag"><a href="/tag/美国/">#美国</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/程序/">#程序</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/rust/">#rust</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/数据/">#数据</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/warning/">#warning</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/代码/">#代码</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/ai/">#ai</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/软件/">#软件</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/linux/">#linux</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/设计/">#设计</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/学习/">#学习</a></button></div><div class="my_movie_list_item shadow p-3 mb-5 bg-white rounded"><button type="button" class="btn btn-link my_tag"><a href="/tag/web2.0/">#web2.0</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/google/">#google</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/设计/">#设计</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/创意/">#创意</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/摄影/">#摄影</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/游戏/">#游戏</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/图片/">#图片</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/软件/">#软件</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/视频/">#视频</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/手机/">#手机</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/广告/">#广告</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/apple/">#apple</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/iphone/">#iphone</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/网站/">#网站</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/免费/">#免费</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/下载/">#下载</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/windows/">#windows</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/微软/">#微软</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/firefox/">#firefox</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/苹果/">#苹果</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/blog/">#blog</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/音乐/">#音乐</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/博客/">#博客</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/wordpress/">#wordpress</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/恶搞/">#恶搞</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/艺术/">#艺术</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/qq/">#qq</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/web/">#web</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/谷歌/">#谷歌</a></button><button type="button" class="btn btn-link my_tag"><a href="/tag/工具/">#工具</a></button></div></div></div></div><div id="my_footer"><div class=""><a href="/tags/">tags</a> <a href="/users/">users</a></div>&copy;2012-2021 diglog.com </div></div></body></html>